chain/chain/src/metrics.rs

```diff
@@ -259,9 +259,10 @@ pub(crate) static STATE_TRANSITION_DATA_GC_CLEARED_ENTRIES: LazyLock<IntCounter>
     });
 
 pub(crate) static STATE_TRANSITION_DATA_GC_TIME: LazyLock<Histogram> = LazyLock::new(|| {
-    try_create_histogram(
+    try_create_histogram_with_buckets(
         "near_state_transition_data_gc_time",
         "Time taken to do garbage collection of state transaction data",
+        exponential_buckets(0.001, 1.6, 20).unwrap(),
```

Comment:
could you please explain your choice for selecting such buckets

Reply:
Discussed offline that the intention of using custom buckets here is to reduce total size of metrics collected. Added a comment and changed buckets accordingly in [d43bb7b](https://github.com/near/nearcore/pull/13175/commits/d43bb7bb9b6aba3e565e8663b2e3503ba437d98a).